<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="7" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Rest Assured Automation Suite" duration-ms="5479" started-at="2020-12-02T11:11:37Z" finished-at="2020-12-02T11:11:42Z">
    <groups>
    </groups>
    <test name="Test API Calls for POST,GET,PUT,Delete" duration-ms="5479" started-at="2020-12-02T11:11:37Z" finished-at="2020-12-02T11:11:42Z">
      <class name="GetRequest.GetData">
        <test-method status="PASS" signature="TestBody()[pri:0, instance:GetRequest.GetData@3bfdc050]" name="TestBody" duration-ms="1521" started-at="2020-12-02T11:11:37Z" finished-at="2020-12-02T11:11:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestBody -->
        <test-method status="PASS" signature="TestResponse()[pri:0, instance:GetRequest.GetData@3bfdc050]" name="TestResponse" duration-ms="276" started-at="2020-12-02T11:11:38Z" finished-at="2020-12-02T11:11:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestResponse -->
      </class> <!-- GetRequest.GetData -->
      <class name="GetRequest.DeleteData">
        <test-method status="PASS" signature="Test1()[pri:0, instance:GetRequest.DeleteData@91161c7]" name="Test1" duration-ms="682" started-at="2020-12-02T11:11:42Z" finished-at="2020-12-02T11:11:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
      </class> <!-- GetRequest.DeleteData -->
      <class name="GetRequest.PutData">
        <test-method status="PASS" signature="Test1()[pri:0, instance:GetRequest.PutData@5c3bd550]" name="Test1" duration-ms="645" started-at="2020-12-02T11:11:41Z" finished-at="2020-12-02T11:11:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
      </class> <!-- GetRequest.PutData -->
      <class name="GetRequest.GetDataSecondWay">
        <test-method status="PASS" signature="TestBody()[pri:0, instance:GetRequest.GetDataSecondWay@1bce4f0a]" name="TestBody" duration-ms="997" started-at="2020-12-02T11:11:39Z" finished-at="2020-12-02T11:11:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestBody -->
        <test-method status="FAIL" signature="TestResponsecodes()[pri:0, instance:GetRequest.GetDataSecondWay@1bce4f0a]" name="TestResponsecodes" duration-ms="417" started-at="2020-12-02T11:11:40Z" finished-at="2020-12-02T11:11:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [203]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [203]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at GetRequest.GetDataSecondWay.TestResponsecodes(GetDataSecondWay.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestResponsecodes -->
      </class> <!-- GetRequest.GetDataSecondWay -->
      <class name="GetRequest.PostData">
        <test-method status="PASS" signature="Test1()[pri:0, instance:GetRequest.PostData@5e3a8624]" name="Test1" duration-ms="916" started-at="2020-12-02T11:11:40Z" finished-at="2020-12-02T11:11:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
      </class> <!-- GetRequest.PostData -->
    </test> <!-- Test API Calls for POST,GET,PUT,Delete -->
  </suite> <!-- Rest Assured Automation Suite -->
</testng-results>
